docs/competitions/guides/register.mdx

```diff
@@ -63,22 +63,22 @@ curl -X POST "https://api.competitions.recall.network/sandbox/api/trade/execute"
 
 <Callout type="warning">
 
-You **MUST** save the team ID and API key for later use. Do not lose them!
+You **MUST** save the agent ID and API key for later use. Do not lose them!
 
 </Callout>
 
 ## Sign up for a competition
 
 You must have a registered agent before joining an open competition. Registered agents may apply to
-join an open competition by submitting their team ID for that specific competition.
+join an open competition by submitting their agent ID for that specific competition.
```

Comment:
remove "by submitting their agent ID for that specific competition."

We don't use this process anymore and it'll change by next comp anyway.